python/cugraph/structure/convert_matrix.py

```diff
@@ -50,9 +50,9 @@ def from_cudf_edgelist(df, source='source', target='target', weight=None):
     G = DiGraph()
 
     if weight is None:
-        G.from_cudf_edgelist(df, source=source, target=target)
+        G.from_cudf_edgelist(df, source=source, destination=target)
```

Comment:
still here :)

Reply:
from_cudf_edgelist of convert_matrix.py is different from the graph function from_cudf_edgelist. All the tests/examples for loading edge lists into a Graph use "G=cugraph.Graph();G.from_cudf_edgelist" instead of "cugraph.from_cudf_edgelist" currently. This function in convert_matrix.py was provided initially to have something similar to networkx.from_pandas_edgelist(df, source, target) which in cugraph is cugraph.from_cudf_edgelist(df, source, target)...which ultimately calls G.from_cudf_edgelist under the hood and is potentially redundant, but it follows nx exactly)

Reply:
Yea I see that's a bit confusing to have 2 `from_cudf_edgelist`. Do we really still need the function in convert_matrix.py?

Reply:
No, in terms of adding edgelist, we already have the necessary function in cugraph as G.from_cudf_edgelist() in place of networkx's G.add_edges_from(). We could remove the one from convert_matrix.py with the tradeoff of not having a cugraph.from_cudf_edgelist similar to nx.from_cudf_edgelist() if it is confusing

Reply:
let's do that

Reply:
removed